sceptre/resolvers/stack_output.py

```diff
@@ -167,3 +167,42 @@ def resolve(self):
         return self._get_output_value(
             dependency_stack_name, output_key, profile
         )
+
+
+class StackOutputExternalRegion(StackOutputBase):
+    """
+    Resolver for retrieving the value of an output of any Stack in any region.
+
+    :param argument: The Stack name, output name, and region to get.
+    :type argument: str in the format ``"<full stack name>::<output key> <region>"``
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(StackOutputExternalRegion, self).__init__(*args, **kwargs)
+
+    def resolve(self):
+        """
+        Retrieves the value of CloudFormation output of the external Stack
+
+        :returns: The value of the Stack output.
+        :rtype: str
+        """
+
+        self.logger.debug(
+            "Resolving external Stack output: {0}".format(self.argument)
+        )
+
+        profile = None
+        arguments = shlex.split(self.argument)
+
+        stack_argument = arguments[0]
+        region = arguments[1]
+
+        # optional profile
+        if len(arguments) > 2:
+            profile = arguments[2]
+
+        dependency_stack_name, output_key = stack_argument.split("::")
+        return self._get_output_value(
+            dependency_stack_name, output_key, profile, region
```

Comment:
When `profile` is set to `None`, `_get_output_value` uses the default profile instead of the stack config's profile. I don't know if this is intended behavior or not.

EDIT: upon further inspection, it appears to be. Is that a good idea for this resolver? I can see it leading to some confusion.